tcms/testcases/forms.py

```diff
@@ -118,6 +118,7 @@ class BaseCaseSearchForm(forms.Form):
     items_per_page = forms.ChoiceField(
         required=False,
         choices=ITEMS_PER_PAGE_CHOICES)
+    text = forms.CharField(required=False)
```

Comment:
FTR not all of these fields (SearchCaseForm technically) are used to render to form on the FE. The query itself is done via JS. 

I'm not even sure we need these forms but this can be tacked later b/c there is a TODO comment for them.

Reply:
I'll test and eventually remove this addition.